homework_5/Methods/GuessWordGame.java

```diff
@@ -1,5 +1,35 @@
 package Methods;
 
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
 public class GuessWordGame {
-    
+    public static void main(String[] args){
+        Scanner scanner = new Scanner(System.in);
```

Comment:
please extract the logic from the main to a separate method. Choose some appropriate name for it, for example, startGame()